src/table/src/TableVirtualBody.tsx

```diff
@@ -304,7 +304,7 @@ export default class TableVirtualBody extends PureComponent<
           scrollOffset={scrollOffset}
           scrollToAlignment={scrollToAlignment}
           renderItem={({ index, style }) => {
-            const child = children[index]
+            const child: any = children[index]
```

Comment:
Any benefit in declaring as `React.ReactNode` instead of `any`?

Reply:
I've been using `any` with the intent that we will try to eradicate them later. This at least signals that we don't really know what the right solution is, but gotta get the compiler happy as we slowly migrate.